internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -135,6 +135,92 @@ func (t *CreateObjectTest) WrappedSucceeds() {
 	ExpectEq(obj, o)
 }
 
+////////////////////////////////////////////////////////////////////////
+// CreateChunkUploader
+////////////////////////////////////////////////////////////////////////
+
+type CreateChunkUploaderTest struct {
+	fastStatBucketTest
+}
+
+func init() { RegisterTestSuite(&CreateChunkUploaderTest{}) }
+
+func (t *CreateChunkUploaderTest) CallsEraseAndWrapped() {
+	const name = "taco"
+
+	// Erase
+	ExpectCall(t.cache, "Erase")(name)
+
+	// Wrapped
+	var wrappedReq *gcs.CreateObjectRequest
+	ExpectCall(t.wrapped, "CreateChunkUploader")(Any(), Any(), Any(), Any()).
+		WillOnce(DoAll(SaveArg(1, &wrappedReq), Return(nil, errors.New(""))))
+
+	// Call
+	req := &gcs.CreateObjectRequest{
+		Name: name,
+	}
+
+	_, _ = t.bucket.CreateChunkUploader(context.TODO(), req, 1, nil)
+
+	AssertNe(nil, wrappedReq)
+	ExpectEq(req, wrappedReq)
+}
+
+func (t *CreateChunkUploaderTest) WrappedFails() {
+	var err error
+	const name = "taco"
+
+	// Erase
+	ExpectCall(t.cache, "Erase")(Any())
+
+	// Wrapped
+	ExpectCall(t.wrapped, "CreateChunkUploader")(Any(), Any(), Any(), Any()).
+		WillOnce(Return(nil, errors.New(name)))
+
+	// Call
+	_, err = t.bucket.CreateChunkUploader(context.TODO(), &gcs.CreateObjectRequest{}, 1, nil)
+
+	ExpectThat(err, Error(HasSubstr(name)))
+}
+
+// func (t *CreateChunkUploaderTest) WrappedSucceeds() {
```

Comment:
Is this supposed to be commented?

Reply:
Why I can't implement this particular test case is in https://github.com/GoogleCloudPlatform/gcsfuse/pull/1417#discussion_r1377425289 . So I had to disable it.

Reply:
Removed the commented out test.